packages/scene-composer/src/components/three-fiber/AnimationComponent/AnimationComponent.tsx

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line import/default
+import React, { Fragment, useEffect, useMemo } from 'react';
+import * as THREE from 'three';
+import { useFrame } from '@react-three/fiber';
+
+import { useSceneComposerId } from '../../../common/sceneComposerIdContext';
+import { ISceneNodeInternal, useEditorState, useViewOptionState } from '../../../store';
+import { IAnimationComponent } from '../../../interfaces';
+
+export const toggleIsAnimationPaused = (
+  AnimationMixer: THREE.AnimationMixer,
+  isPaused: boolean,
+  AnimationList: string[],
+) => {
+  const object = AnimationMixer.getRoot() as THREE.Object3D;
+  AnimationList.forEach(function (clip) {
+    const animation = THREE.AnimationClip.findByName(object.animations, clip);
+    if (AnimationMixer.clipAction(animation)) {
+      AnimationMixer.clipAction(animation).paused = isPaused;
+    }
+  });
+};
+
+interface AnimationComponentProps {
+  component: IAnimationComponent;
+  node: ISceneNodeInternal;
+}
+
+const AnimationComponent: React.FC<AnimationComponentProps> = ({ component, node }) => {
+  const sceneComposerId = useSceneComposerId();
+  const isGlobalAnimationPaused = !useViewOptionState(sceneComposerId).componentVisibilities['Animation'];
+  const [scene, setScene] = React.useState<THREE.Object3D>();
+  const { getObject3DBySceneNodeRef } = useEditorState(sceneComposerId);
+  const object = getObject3DBySceneNodeRef(node.ref);
+  const currentAnimations = useMemo(() => {
+    const currentAnimations: any[] = [];
+    if (object) {
+      setScene(object.getObjectByName('Scene'));
```

Comment:
Needs comment explaining magic string for future programmers.

Reply:
Also probably should be a const so we can update the definition once and have it update everywhere.